cf-file-scanner/deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Script to prepare and deploy function.
+
+set -e
+
+if (( $# != 1 )); then
+  echo 'usage: deploy.sh <GCP_PROJECT>'
+  exit 1
+fi
+
+PROJECT_ID="$1"
+SECRET_ID='cf-file-scanner'
+TOPIC_NAME='cf-file-scanner'
+CF_NAME='cf-file-scanner'
+SA_ACCT="cf-file-scanner@$PROJECT_ID.iam.gserviceaccount.com"
+
+./build-clamav.sh
+
+echo 'Rendering config file...'
+gcloud --project=$PROJECT_ID secrets versions access latest \
+  --secret=$SECRET_ID > application.conf
+
+echo 'Uploading function...'
+echo 'Note: it might take a few minutes to upload clamav binaries.'
+
+echo ''
+gcloud --project=$PROJECT_ID functions deploy $CF_NAME \
+  --entry-point org.broadinstitute.ddp.cf.FileScanner \
+  --trigger-topic $TOPIC_NAME \
+  --service-account $SA_ACCT \
+  --region us-central1 \
+  --runtime java11 \
+  --memory 2048MB \
+  --timeout 5m \
```

Comment:
I tried running a scan on some large files and it ran fine, well within a minute, so I think 5 minute timeout should be good.

12+MB file:
![Screen Shot 2021-02-04 at 2 35 04 PM](https://user-images.githubusercontent.com/5713833/106948395-a5d4ab00-66f9-11eb-9596-7749dfa1449c.png)

84+MB file:
![Screen Shot 2021-02-04 at 2 37 47 PM](https://user-images.githubusercontent.com/5713833/106948486-c43aa680-66f9-11eb-9da0-e8f17032e5b1.png)

Reply:
LGTM! Great analysis here